pages/your_domain/human_data.md

```diff
@@ -44,15 +44,18 @@ When working with human data, you must follow established research ethical guide
 * Have you got an **ethical permit** for your research project?
   * To get an ethical permit, you have to apply for an **ethical review** by an **ethical review board**.
     - The legislation that governs this differs between countries. Do seek advice from your research institute.
-  * In most cases, you should get **informed consents** from your research subjects.
+  * The [Global Alliance for Genomics and Health (GA4GH)](https://www.ga4gh.org) has recommendations for these issues in their [GA4GH regulatory and ethical toolkit](https://www.ga4gh.org/genomic-data-toolkit/regulatory-ethics-toolkit/), see for instance the [Consent Clauses for Genomic Research](https://drive.google.com/file/d/1O5Ti7g7QJqS3h0ABm-LyTe02Gtq8wlKM/view?usp=sharing).
+* The acquisition of data must be legal.
+  * **Receiving data/samples directly from data subjects** requires in most cases **informed consents**.
     - An informed consent is an agreement from the research subject to participate in and share personal data for a particular purpose. It shall describe the purpose and any risks involved (along with any mitigations to minimise those risks) in such a way that the research subject can make an informed choice about participating. It should also state under what circumstances the data can be used for the initial purpose, as well as for later re-use by others.
       - Consider describing data use conditions using a machine-readable formalised description such as the Data Use Ontology [DUO](https://github.com/EBISPOT/DUO). This will greatly improve the possibilities to make the data FAIR later on.
```

Comment:
I think the link for DUO mentioned here is https://github.com/EBISPOT/DUO not necessary, as it is mentioned later on under solutions. If we want to add link here then it should be pointed to a general resource on DUO.

Reply:
fixed in https://github.com/elixir-europe/rdmkit/pull/1266/commits/e405830f63ec76171f350e55978361ee4f566f13